artillery/.gitignore

```diff
@@ -0,0 +1,2 @@
+test-run-report.json
+test-run-report.json.html
```

Comment:
Does this need the .env file too?

Reply:
No, the env file has been deleted in the latest commit as its not used anymore

Reply:
Ah I thought I'd deleted it before running this version... guess not!